app/models/assignment/editor.rb

```diff
@@ -91,7 +91,9 @@ def update_attribute_for_all_assignment_repos(attribute:, change:)
     end
 
     def deadline_updated?
-      @options[:deadline] != @assignment.deadline&.deadline_at
+      return false unless @options[:deadline]
+      new_deadline_at = DateTime.strptime(@options[:deadline], '%m/%d/%Y %H:%M %z').utc
```

Comment:
We also use this format string in Deadline::Factory. Would you be able to refactor them both out to a shared constant somewhere incase we ever change this format?